subsys/bootloader/provision/provision_flash.c

```diff
@@ -11,16 +11,27 @@
 #include <nrf.h>
 #include <assert.h>
 #include <pm_config.h>
+#include <nrfx_nvmc.h>
+
 
 typedef struct {
 	u32_t s0_address;
 	u32_t s1_address;
 	u32_t num_public_keys;
-	u32_t pkd[1];
+	u32_t key_data[1];
 } provision_flash_t;
 
+#if defined(CONFIG_SOC_NRF9160)
+
```

Comment:
Remove surrounding newlines to match style of the `elif` case

Reply:
Moved some newlines around